community/kernel/src/main/java/org/neo4j/kernel/impl/storemigration/participant/StoreMigrator.java

```diff
@@ -371,6 +371,8 @@ public boolean parallelRecordReadsWhenWriting()
                     readAdditionalIds( lastTxId, lastTxChecksum, lastTxLogVersion, lastTxLogByteOffset );
 
             // We have to make sure to keep the token ids if we're migrating properties/labels
+            // Use the standard importer, instead of loading from BatchImporterFactory because an aborted store migration
+            // will be restarted from scratch anyway
```

Comment:
That's true that there is no feature driven reason to use edition based importer in this place right now, but still, maybe it makes sense to do a switch now so any future edition based improvements can be used automatically?

Reply:
Perhaps you're right, I'm OK with doing this change.